src/ExpressionEvaluator/Core/Test/ResultProvider/ReflectionUtilities.cs

```diff
@@ -41,19 +41,21 @@ internal static AssemblyLoadContext LoadAssemblies(params Assembly[] assemblies)
 
         internal static Assembly[] GetMscorlib(params Assembly[] additionalAssemblies)
         {
-            var builder = ArrayBuilder<Assembly>.GetInstance();
-            builder.Add(typeof(object).Assembly); // mscorlib.dll
-            builder.AddRange(additionalAssemblies);
-            return builder.ToArrayAndFree();
+            return
+            [
```

Comment:
>[ [](http://example.com/codeflow?start=12&length=1)

Out of curiosity, what does this now generate under the hood?

Reply:
I guess I'm more curious about the change in InstallPackageParentCodeAction.cs as it creates an ImmutableArray from this.

Reply:
The codegen is great:

![image](https://github.com/dotnet/roslyn/assets/4564579/8d71af7b-7e28-47e4-aef0-c8bd3c895f14)

https://sharplab.io/#v2:EYLgtghglgdgPgAQEwEYCwAoBAGABAlAOgCUBTAMwBtSBjAFygHsYBuTHfIgYUcuvqYwAzoQCSYMAFc6EYNTYZ2AZnxJcXXAG9MuXblh1SAJxgRKnAGy5xUmXNIBBI0YgBPADwOhQ0mDmuAPlwAcVI6AFkhGkYjSihgAAoABwgXMCFcLx8/SlcAbQBdXAgAExKoBmYzLN85KFIhAEodPW0MPQ78AHZcPJbOgbpXJNJGcgTGYAArWjpGwhqc1wAaXAB6Ndx06Nj4whK+foHjwkJisorBau9auIblo46ChQ6AX0xXoA===

Reply:
That is beautiful. Is  ImmutableCollectionsMarshal net8 only?

Reply:
I don't believe so. You just get it if you have an updated/modern System.Collections.Immutable dll.